cmd/batch.go

```diff
@@ -51,3 +62,14 @@ func batchCmd(configPath, logLevel, logFormat *string) *cobra.Command {
 
 	return cmd
 }
+
+func cloneOrPull(git *git.Git, project project.Project) error {
```

Comment:
Do we want to clone? Some projects have >50GB. It is risky IMHO (or needs to update the chart deployment for a specific config).
Maybe we can just fetch the latest commit (like CI behavior by default). Wdyt?

Reply:
It is a good idea, I just reused what we do for the daemon command for now

Reply:
I see that shallow clone is disabled [here](https://github.com/qonto/standards-insights/blob/c2c8d881ead843deb6e5a97b98140673993c1bc9/internal/git/git.go#L60), do you remember why @mcorbin ?

Reply:
I wasn't sure about the impact: https://github.com/CocoaPods/CocoaPods/issues/4989#issuecomment-193772935